react/features/base/environment/environment.js

```diff
@@ -81,6 +83,18 @@ export function isSupportedBrowser() {
     return isMobileBrowser() || JitsiMeetJS.isWebRtcSupported();
 }
 
+/**
+ * Returns whether or not the current environment is a supported
+ * browser on a mobile device.
+ *
+ * @returns {boolean}
+ */
+export function isSupportedMobileBrowser() {
+    return (Platform.OS === 'android' && browser.isChrome())
+        || (Platform.OS === 'android' && browser.isFirefox())
+        || (Platform.OS === 'ios' && browser.isSafari());
```

Comment:
Do we know if web views are detected as Safari. IIRC WebRTC does not work in them.

Reply:
I don't know, I would like to incorporate that check in browser-detection in js-utils which is what I will be working on next.